src/translations/data.json

```diff
@@ -2150,7 +2150,7 @@
     "isv": "Russky",
     "ru": "Русский",
     "uk": "Російська",
-    "be": "Руская",
+    "be": "Расейская",
```

Comment:
Are you sure about it? On all online resources (translators/dictionaries), we found that right "руская мова" maybe you mixed up with Ukrainian, there "російська мова"?

Reply:
@Zmicier21 can you please review this?

Reply:
[](url)Yes, I am sure, because modern Belarusian users are more prevalent in this variant, which is fixed by dictionaries. The Belarusian language is the widest than it may seem.
```suggestion
    "be": "Расейская",
```